### PR TITLE
rails 5.2.0 support

### PR DESCRIPTION
**Pull request by egze**
specs seem to run fine

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,3 +42,10 @@ matrix:
       rvm: 2.1
     - gemfile: gemfiles/rails5.1.gemfile
       rvm: 2.2
+
+    - gemfile: gemfiles/rails5.2.gemfile
```
Comment (grosser) [inline .travis.yml]: FYI can also drop 2.0/2.1

**Comment by grosser** [comment]
look into https://dev.to/evilmartians/rails-aftercommit-everywhere--4j9g too, it seems like a much lighter alternative

File: .travis.yml (hunk 1 above)
Comment (egze) [reply .travis.yml]: done

**Comment by egze** [comment]
@grosser cheers for the alternative. I'm currently upgrading an app to 5.2.0 and would just want it to run with existing gems. But I'll add a todo to update to `isolator`.

File: .travis.yml (hunk 1 above)
Comment (grosser) [reply .travis.yml]: need to remove them from the `rvm` above and bump ruby version in gemspec

Comment (egze) [reply .travis.yml]: Not sure how to "bump ruby version in gemspec". What do you mean by that?

**Comment by grosser** [comment]
s.required_ruby_version = '>= 2.2.0'

**Comment by grosser** [comment]
needs a `before_install: gem install bundler`

**Comment by egze** [comment]
Success! 🍏

**Comment by egze** [comment]
Thanks for your time!

**Comment by grosser** [comment]
thanks for the PR :)
v0.5.0

**Pull request merged**